### ISSUE TITLE
The ontology build process must allow external control of resource locations

### ISSUE THREAD

**Issue by kltm** [geneontology/go-ontology#18870]
Currently, an incorrect location is hard-coded into the ontology build process:
https://github.com/geneontology/go-ontology/blob/79299ea5aafdd1fad735c9bf3bcbf86a77c2a044/src/ontology/Makefile#L151-L155

While the best "general" location of this file it going to be http://snapshot.geneontology.org/metadata/db-xrefs.legacy or http://current.geneontology.org/metadata/db-xrefs.legacy , if this is running in the pipeline, the URL needs to be something like http://skyhook.berkeleybop.org/A/SERIES/OF_RESOLVED_VARIABLES/metadata/db-xrefs.legacy .

To accomplish this, the Makefile should have external resources and references to other GO build products as variables (with a sensible default).

For more context, see https://github.com/geneontology/pipeline/issues/162

**Comment by pgaudet** [comment]
@kltm 

Is this still an issue?

**Comment by pgaudet** [comment]
@kltm says The ontology build should only be using data internally created within the pipeline, not snapshot

**Comment by ValWood** [comment]
Is this still an issue? Who should it be assigned to?

**Comment by ValWood** [comment]
@kltm  is this still an issue?
Thanks
Val

**Comment by kltm** [comment]
@ValWood Yes, this is still outstanding. I don't think this is a priority for the moment, but the work would most likely be carried out at some point by (mostly) @balhoff and (a wee bit) myself.
